server_environment.yml

```diff
@@ -9,15 +9,15 @@ dependencies:
 - mesalib
 - ffmpeg
 - vtk
-- numpy
 - scipy
 - traits
 - pip:
   - mne
+  - numpy<1.19.0
```

Comment:
This is weird, should probably just as the version constraint above, or do everything in pip rather than first installing NumPy above via conda then killing it by installing one over it in pip

Reply:
(scipy depends on NumPy, so removing just NumPy from the conda list is essentially the same as leaving it in)

Reply:
The kernel dies so I will move scipy and try again

Reply:
FWIW I'm sure this solution works, is just not clean. It also might do some weird things with MKL at some point (for example if we switch to the base channel rather than conda forge).

Basically I'm suggesting either pin above (easier if conda actually supports or), or move everything that requires NumPy to the pip section (harder to ensure is implemented and maintained properly)

Reply:
Almost everything uses pip. We need conda mainly because of mesalib and vtk (not sure it's needed anymore).

Reply:
Actually, `vtk` also requires `numpy` so it should also be moved. I'll push a commit